### PR TITLE
Bump explore and all dependencies to versions supporting monocle 3 and scalajs-react 2.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/main/scala/explore/observationtree/obsTree.scala [hunk 1]
```diff
@@ -5,26 +5,35 @@ package explore.observationtree
 
 import explore.model.ObsSummary
 import monocle.Lens
-import monocle.macros.Lenses
+import monocle.macros.GenLens
```
Comment (cquiroz) [inline explore/src/main/scala/explore/observationtree/obsTree.scala:8]: Maybe we should use `Focus` everywhere

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/observationtree/obsTree.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/observationtree/obsTree.scala:8]: Oh, I missed we were using `GenLens`. Is there a difference with `Focus`, or just the new name?

**Comment by rpiaggio** [comment]
I should mention that this PR attempts to be a minimal set of changes to allow migration (and unblock the new features).

However, we still have some overlap between crystal and scalajs-react 2, albeit with slightly different approaches and with the latter probably enables simpler ways of doing things.

I'm going to look at these next and try to coalesce things into scalajs-react, removing from crystal what's no longer necessary.

**Pull request merged**